EIPS/eip-6963.md

```diff
@@ -39,27 +39,72 @@ Wallet: A user agent that manages keys and facilitates transactions with Ethereu
 
 Decentralized Application (DApp): A web page that relies upon one or many Web3 platform APIs which are exposed to the web page via the Wallet.
 
-Provider Discovery Library: A library or piece of software that assists a DApp to interact with the Wallet. 
+Provider Discovery Library: A library or piece of software that assists a DApp to interact with the Wallet (e.g. Web3Modal, wagmi, RainbowKit, Web3-Onboard, ConnectKit, etc).
 
 ### Provider Info
 
-Each wallet provider will be announced with the following interface `EIP6963ProviderInfo`. The values in the `EIP6963ProviderInfo` MUST be used as follows:
+Each Wallet Provider will be announced with the following interface `EIP6963ProviderInfo`. The values in the `EIP6963ProviderInfo` MUST be used as follows:
 
-- **`uuid`** - a globally unique of the wallet provider that MUST be (UUIDv4[^rfc4122] compliant) to uniquely distinguish different [EIP-1193](./eip-1193.md) provider sessions. The cryptographic uniqueness provided by UUIDv4[^rfc4122] guarantees that two independent `EIP6963ProviderInfo` objects can be separately identified.
-- **`name`** - A human-readable local alias of the wallet provider to be displayed to the user on the DApp. (e.g. `DopeWalletExtension` or `AwesomeWallet`)
-- **`icon`** - A URI[^rfc3986] pointing to an image. Icon images MUST be square with 96x96px minimum resolution
+- **`uuid`** - a universally unique identifier of the Wallet Provider that MUST be UUIDv4[^rfc4122] compliant and thus MUST BE nondeterministic between sessions.
+- **`providerId`** - a unique identifier for the Wallet Provider that MUST be deterministic between sessions.
```

Comment:
It's determinsitic between sessions if the identifier is identical between sessions. Should we define what "determinism" and "session" means Definitions?

Reply:
Probably if we want to distinguish between what they mean. Having clear definitions for things like this can make it easier for the readers and implementors to understand why each one is necessary.

Reply:
Yep agree. We could also probably rename "session" to "page instance" or something more specific to the browser.